src/Components/Search/NewSearch/Mobile/Overlay.tsx

```diff
@@ -34,14 +36,22 @@ interface OverlayProps {
 
 export const Overlay: FC<OverlayProps> = ({ viewer, relay, onClose }) => {
   const { t } = useTranslation()
+  const tracking = useTracking()
   const inputRef = useRef<HTMLInputElement | null>(null)
   const [selectedPill, setSelectedPill] = useState<PillType>(TOP_PILL)
   // TODO: Parse value from url
   const [inputValue, setInputValue] = useState("")
   const disablePills = !shouldStartSearching(inputValue)
+  const [fetchCounter, setFetchCounter] = useState(0)
 
   useEffect(() => {
     inputRef.current?.focus()
+
+    tracking.trackEvent({
+      action_type: ActionType.focusedOnSearchInput,
+      context_module: selectedPill.analyticsContextModule,
+    })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why disable linter: we want to track focus on the input only once, at the same time we want to send current pill's context info. When selecting another pill - this effect shouldn't be executed again

Reply:
For comments like this, you would want to leave them in the code itself.

Reply:
done